app/views/events/index.html.erb

```diff
@@ -16,25 +15,26 @@
       <th>End time</th>
       <th>Duration</th>
       <th>Status</th>
+      <th>Template</th>
       <th></th>
     </tr>
   </thead>
-
   <tbody>
     <% @events.each do |e| %>
-      <tr>
+      <tr class='<%= column_color_class(e) %>'>
         <td><%= link_to e.title, e %></td>
         <td><%= e.location %></td>
         <td><%= e.start_time %></td>
         <td><time>
-          <%= e.start_time.strftime('%a %b %d %k:%M') if e.start_time  %>
+          <%= e.start_time.strftime('%a %m/%d/%Y %I:%M %p') if e.start_time  %>
```

Comment:
You've changed the format of the time here. Do you need it in this format ? The end users actually prefer the other. They don't need the year (saving 10 chars when they're viewing on their phone) and they all use a 24 hour clock, eliminating the AM/PM. Finally, you can get into an issue with people parsing the month/day incorrectly, due to differing cultures. Consider 1/3/2017. Is it January 3rd, or March 1st ?
For comparision:
New format => Fri 09/29/2017 06:30 PM
Old Format => Fri Sep 29 18:30
If you look back through the history, you can see the evolution to this time format.

Reply:
@kgf I will change this back to its original form, It was helpful for me to read during debugging.

Reply:
I am interested; what made it difficult ? I could see where some folks will miss the year. But how to properly delimit it ?
Fri Sep 29 '17 1830   ?
Fri 09/29/17 1830    ?

Reply:
It was mostly because the year was not present, you can use '%y' to delimit the year to the last 2 digits of the century. eq. to (year % 100)

Reply:
I listed two examples just above your comment. Would you use one of those ? Something else ?
I like the `Sep` for the month, but it doesn't seem to _go_ with the year somehow. Or maybe it's just me....

Reply:
Yea, the first one you suggested looks good to me. The only thing I would add is commas after the weekday and day of month. Do you agree?  - Fri, Sep 29, 17 1830

Reply:
OK, go with this, we'll see what folks think
`Fri, Sep 29, 17 1830`

Reply:
Sounds good, any other assessments, or should I commit these changes?